src/algs/ags/solver.cc

```diff
@@ -58,7 +58,7 @@ namespace
           right[i] = mRightBound[i];
         }
       }
-      int GetOptimumPoint(double* y) const {return 0;}
+      int GetOptimumPoint(double* y) const {(void) y; return 0;}
```

Comment:
I think you can simply use `GetOptimumPoint(double* ) const {return 0;}`

Reply:
for C++ this is correct, but in C that won't work (for each compiler), I'll change it here.